.github/workflows/publish-spark-metadata-client.yaml

```diff
@@ -37,14 +46,12 @@ jobs:
           password=${OSSRH_TOKEN}
           EOS
         env:
-          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           OSSRH_USERNAME: ${{ secrets.OSSRH_USERNAME }}
           OSSRH_TOKEN: ${{ secrets.OSSRH_TOKEN }}
+
       - name: Publish artifacts
```

Comment:
Did you check this one?

Reply:
Not yet - from the docs it supposes to use the machine credentials, and we assumed role.